tools/cockpit.spec

```diff
@@ -456,6 +459,17 @@ test -f %{_bindir}/firewall-cmd && firewall-cmd --reload --quiet || true
 
 %if 0%{?rhel} == 0
 
+%package kdump
+Summary: Cockpit user interface for kernel crash dumping
+Requires: %{name}-bridge >= %{required_base}
+Requires: %{name}-shell >= %{required_base}
+BuildArch: noarch
+
+%description kdump
+The Cockpit component for configuring kernel crash dumping.
+
+%files kdump -f kdump.list
+
```

Comment:
The debian files need to build and install cockpit-kdump ... if that indeed is to be delivered there. It seems to be the case since you built a new Debian image.